app.js

```diff
@@ -1,6 +1,21 @@
-let count = 0;
-document.querySelector('.my-pic').addEventListener('click', function(){
-    count++;
-    document.getElementById('btn').innerHTML= count;
-    localStorage.setItem('count','count');
+document.addEventListener('DOMContentLoaded', getCount);
+let counter1 = 0;
+let counter2 = 0;
+document.querySelector('.my-pic1').addEventListener('click', function(){ 
+    counter1++;
+    localStorage.setItem('counter1',counter1);

```

Comment:
instead of setting two different local storage values, you can instead create an object and update it.
After every click, you can save the updated object to the local storage

Reply:
Thank you sir. I have updated the code in the recent commit.